build/commands/lib/config.js

```diff
@@ -914,9 +923,6 @@ Object.defineProperty(Config.prototype, 'defaultOptions', {
     if (this.use_goma && this.gomaServerHost) {
       env.CC_WRAPPER = path.join(this.gomaDir, 'gomacc')
       env.GOMA_SERVER_HOST = this.gomaServerHost
-      // env.NINJA_REMOTE_NUM_JOBS = this.gomaJValue
```

Comment:
So NINJA_REMOTE_NUM_JOBS not needed after all?

Reply:
@goodov it is (or at least was used at some point) by android https://android.googlesource.com/platform/build/+/c1cba69%5E%21/

Reply:
I saw that old change, but I don't see `NINJA_REMOTE_NUM_JOBS ` is used anywhere now.